### ISSUE TITLE
SVG label in wrong place

### ISSUE THREAD

**Issue by jvine** [sul-dlss/earthworks#671]
- [x] Re #657 - the `aria-label` has been added to the `<span>` element, but it should be on the `<svg>`. 

```
<span class="blacklight-icons blacklight-icon-image" aria-label="Image"><!--?xml version="1.0"?-->
<svg viewBox="0 0 30 28" role="img"><title>Image</title>
```

- [ ] Also, adding `focusable="false"` removes a bit of noise (in VO at least). (Recommended by ARC Toolkit validator.)

After:
 
```
<span class="blacklight-icons blacklight-icon-image"><!--?xml version="1.0"?-->
<svg viewBox="0 0 30 28" role="img" aria-label="Image" focusable="false">
```

**Comment by jvine** [comment]
Also note that the Stanford S `<svg>` still includes `aria-labelledby="aria-label-stanford"` - shd be deleted.

**Comment by mejackreed** [comment]
@camillevilla likely related to (and will want to do together with) #662

**Comment by camillevilla** [comment]
#716 addresses the `aria-label` issue. `focusable="false"` might need further analysis before proposing a change upstream.

**Comment by thatbudakguy** [comment]
Tested with `focusable="false"` in Safari 16.4; it didn't make a difference to how VO read the items in the heading — in both cases we got "Image, image" followed by the next icon.

That's a little confusing, but the situation gets better in geoblacklight v4, where this category is renamed in the schema to "Imagery", so it doesn't sound quite as much like VO is repeating itself. There, you'd hear something like:
> "[title of the record], Imagery (image), Stanford (image), Public Content (image)"

...which doesn't seem too bad.

Closing since I don't think there's any obvious work that can be done to make the current situation better (other than upgrading).
